book/quick_tour.md

```diff
@@ -1,53 +1,233 @@
+Notice that we didn't pass commandline arguments or switches to [`ls`](/commands/docs/ls.md). Instead, we used Nushell's built-in [`sort-by`](/commands/docs/sort-by.md) command to sort the _output_ of the `ls` command. Then, to see the largest files on top, we used [`reverse`](/commands/docs/reverse.md) on the _output_ of `sort-by`.
+
+::: tip Cool!
+If you compare the sort order closely, you might realize that the data isn't sorted alphabetically. It's not even sorted by the _numerical_ values. Instead, since the `size` column is a [`filesize` type](./types_of_data.md#file-sizes), Nushell knows that `1.1 KiB` (kibibytes) is larger than `812 B` (bytes).
+:::
+
+# Finding Data Using the `where` Command
+
+Nu provides many commands that can operate on the structured output of the previous command. These are usually categorized as "Filters" in Nushell.
+
+For example, we can use [`where`](/commands/docs/where.md) to filter the contents of the table so that it only shows files over 10 kilobytes:
+
+```nu
+ls | where size > 10kb
+# => ╭───┬─────────────────┬──────┬───────────┬───────────────╮
+# => │ # │      name       │ type │   size    │   modified    │
+# => ├───┼─────────────────┼──────┼───────────┼───────────────┤
+# => │ 0 │ CONTRIBUTING.md │ file │  11.0 KiB │ 5 months ago  │
+# => │ 1 │ Cargo.lock      │ file │ 194.6 KiB │ 2 minutes ago │
+# => │ 2 │ README.md       │ file │  12.0 KiB │ 16 hours ago  │
+# => │ 3 │ toolkit.nu      │ file │  20.0 KiB │ 16 hours ago  │
+# => ╰───┴─────────────────┴──────┴───────────┴───────────────╯
+```
+
+## More Than Just Directories
+
+Of course, this isn't limited to the `ls` command. Nushell follows the Unix philosophy where each command does one thing well and you can typically expect the output of one command to become the input of another. This allows us to mix-and-match commands in many different combinations.
+
+Let's look at a different command:
+
+```nu:no-line-numbers
+ps
+# => ╭───┬──────┬──────┬───────────────┬──────────┬──────┬───────────┬─────────╮
+# => │ # │ pid  │ ppid │     name      │  status  │ cpu  │    mem    │ virtual │
+# => ├───┼──────┼──────┼───────────────┼──────────┼──────┼───────────┼─────────┤
+# => │ 0 │    1 │    0 │ init(void)    │ Sleeping │ 0.00 │   1.2 MiB │ 2.2 MiB │
+# => │ 1 │    8 │    1 │ init          │ Sleeping │ 0.00 │ 124.0 KiB │ 2.3 MiB │
+# => │ 2 │ 6565 │    1 │ SessionLeader │ Sleeping │ 0.00 │ 108.0 KiB │ 2.2 MiB │
+# => │ 3 │ 6566 │ 6565 │ Relay(6567)   │ Sleeping │ 0.00 │ 116.0 KiB │ 2.2 MiB │
+# => │ 4 │ 6567 │ 6566 │ nu            │ Running  │ 0.00 │  28.4 MiB │ 1.1 GiB │
+# => ╰───┴──────┴──────┴───────────────┴──────────┴──────┴───────────┴─────────╯
+```
+
+You may be familiar with the Linux/Unix `ps` command. It provides a list of all of the current processes running in the system along with their current status. As with `ls`, Nushell provides a cross-platform, built-in [`ps` command](/commands/docs/ps.md) that returns its results as structured data.
```

Comment:
Maybe this would be a good place for a tip mentioning you can still access the external `ps` with `^ps` (and also introduce the internal/external terminology?)

Reply:
Thanks - good point, and added in the latest commit.